htdocs/okapi/meta.php

```diff
@@ -0,0 +1,6 @@
+<?php
+return [
+    'version_number' => 1656,
+    'git_revision' => '429977b75fe260aa3aa240e8005edb84524f2354
+',
+];
```

Comment:
There is a line feed in the git_revision value.

Reply:
Fixed